documentation/modules/ROOT/pages/connectors/vitess.adoc

```diff
@@ -1722,6 +1722,9 @@ You can configure the connector to skip the following types of operations:
 
 Set the value to `none` if you do not want the connector to skip any operations.
 
+The {prodname} Vitess connector emits `truncate` events only to its internal schema history topic and not data change topics.
```

Comment:
Vitess connector actually does not have any internal schema history topic. It constructs schema entirely from the vstream events from vitess.

Reply:
@twthorn I've removed the error. Let me know if this looks ok now. Thanks.